t/t0002-request.t

```diff
@@ -106,14 +106,14 @@ test_expect_success 'request: rpc test client works with no request payload' '
 		test -s attr.list.out
 '
 test_expect_success 'request: rpc test client works with request payload' '
-	$jq -j -c -n  "{name:\"rank\"}" | $RPC attr.get >attr.get.out &&
+	jq -j -c -n  "{name:\"rank\"}" | $RPC attr.get >attr.get.out &&
 		test -s attr.get.out
 '
 test_expect_success 'request: rpc test client handles expected failure' '
 	$RPC attr.get 71 </dev/null
 '
 test_expect_success 'request: rpc test client handles expected failure other than EPROTO' '
-	$jq -j -c -n  "{name:\"noexist\"}" | $RPC attr.get 2
```

Comment:
Commit message typo: restult

Also the statement that tests use `$jq` due to when `jq` was not a requirement isn't strictly true, it is just sometimes useful to look up the path to a program once when you know it will be called many times. However, it seems silly for these tests so I agree with the general approach. However, for a more accurate commit message, I'd say something more along the lines of

> Some tests execute `Jq(1)` via the `$jq` variable set in `t/sharness.d/flux-sharness.sh`, while many just execute `jq` from `PATH` directly. For consistency and to match the use of most other programs in sharness tests, drop the use of the `$jq` variable.